tests/target/legacy-control-style-wrap.rs

```diff
@@ -0,0 +1,12 @@
+// rustfmt-max_width: 100
+// rustfmt-control_style: Legacy
+// Ensure legacy control style produces correct wrapping
+
+fn main() {
+    let aaaaaaaaaa_bbbbbb_cccccc_ddddd = if aaaaaaa_bbbb.cc_dddd() &&
```

Comment:
I think in general we prefer to put the opening brace on the next line before breaking the condition.